src/docs/index.rst

```diff
@@ -119,30 +105,53 @@ Serac.
-developing large codes. To specify which compiler to use add the compiler spec to the ``--spec`` Uberenv
-command line option. On TOSS3, we recommend and have tested ``--spec=%clang@4.0.0``.  More compiler specs
+developing large codes. To specify which compiler to use add the compiler specification to the ``--spec`` Uberenv
+command line option. On TOSS3, we recommend and have tested ``--spec=%clang@9.0.0``.  More compiler specs
 can be found in the Spack compiler files in our repository: 
 ``scripts/uberenv/spack_configs/<System type>/compilers.yaml``.
 
-Some helpful uberenv options:  
+We currently regularly test the following Spack configuration files:
+
+* Linux Ubuntu 18.04 (via Windows WSL 1)
+* Linux Ubuntu 20.04 (via Windows WSL 2)
+* TOSS 3 (On Quartz at LC)
+* BlueOS (On Lassen at LC)
+
+To install Serac on a new platform, it is a good idea to start with a known Spack configuration directory
+(located in the Serac repo at ``scripts/uberenv/spack_configs/<platform>``). The ``compilers.yaml`` file
+describes the compilers and associated flags required for the platform and the ``packages.yaml`` file 
+describes the low-level libraries on the system to prevent Spack from building the world. Documentation on 
+these configuration files is located in the `Spack docs <https://spack.readthedocs.io/en/latest/configuration.html>`_.
 
-* ``--spec=+debug``
-* ``--spec=+glvis``
-* ``--spec=%clang@4.0.0``
-* ``--spec=%clang@4.0.0+debug``
-* ``--spack-config-dir=<Path to spack configuration directory>``
-* ``--prefix=<Path to uberenv build directory (defaults to ./uberenv_libs)>``
+Some helpful uberenv options include :  
 
-If you already have a spack instance you would like to reuse, you can do so changing the uberenv
-command as follow:
+* ``--spec=+debug`` (build the MFEM and Hypre libraries with debug symbols)
+* ``--spec=+glvis`` (build the optional glvis visualization library)
+* ``--spec=+caliper`` (build the `Caliper performance profiling library <https://github.com/LLNL/Caliper>`_)
+* ``--spec=+devtools`` (also build the devtools with one command)
+* ``--spec=%clang@9.0.0`` (build with a specific compiler as defined in the ``compiler.yaml`` file)
+* ``--spack-config-dir=<Path to spack configuration directory>`` (use specific Spack configuration files)
+* ``--prefix=<Path> (defaults to ./uberenv_libs)>`` (build and install the dependencies in a particular location)
+
+The modifiers to the Spack specification ``spec`` can be chained together, e.g. ``--spec=%clang@9.0.0+debug+glvis+devtools``.
+
+If you already have a Spack instance from another project that you would like to reuse, 
+you can do so by changing the uberenv command as follows:
 
 .. code-block:: bash
 
    $ python scripts/uberenv/uberenv.py --upstream=</path/to/my/spack>/opt/spack
 
+Building Serac's Dependencies by Hand
+-------------------------------------
+
+To build Serac's dependencies by hand, a ``host-config`` (CMake configuration file) is required by the BLT 
```

Comment:
not technically required.. "strongly encouraged" since you would have to specify all of it on the command line otherwise..